### PR TITLE
Add Telegram message threads (topics) support

### PR DESCRIPTION
**Pull request by NeonSludge**
This PR adds support for sending alerts to [Telegram group threads/topics](https://telegram.org/blog/topics-in-groups-collectible-usernames/ru?setln=en#topics-in-groups). This is entirely optional and doesn't introduce any breaking changes to existing Robusta installations that send alerts to regular Telegram groups or channels.

This change has been manually tested in a real-world scenario with multiple Telegram sinks.

### CODE REVIEW

File: src/robusta/core/sinks/telegram/telegram_client.py [hunk 1]
```diff
@@ -9,14 +9,16 @@
...
+    def __init__(self, chat_id: int, thread_id: int, bot_token: str):
         self.chat_id = chat_id
+        self.thread_id = thread_id
         self.bot_token = bot_token
 
     def send_message(self, message: str, disable_links_preview: bool = True):
         url = f"{TELEGRAM_BASE_URL}/bot{self.bot_token}/sendMessage"
         message_json = {
             "chat_id": self.chat_id,
+            "message_thread_id": self.thread_id,
```
Comment (arikalon1) [inline src/robusta/core/sinks/telegram/telegram_client.py:21]: Just to be sure, if the `thread_id` is `None`, does this fallback to the default behavior? 
Did you test it with an empt `thread_id`?

**Review by arikalon1 — COMMENTED** [review]
Thanks for your contribution @NeonSludge 

It sounds very useful 
Left one small question in the comments

File: src/robusta/core/sinks/telegram/telegram_client.py (hunk 1 above)
Comment (NeonSludge) [reply src/robusta/core/sinks/telegram/telegram_client.py:21]: Yep, I've successfully tested this with an existing Robusta configuration that contains several Telegram sinks without the `thread_id` parameter set.

Comment (arikalon1) [reply src/robusta/core/sinks/telegram/telegram_client.py:21]: Thank you!

**Review by arikalon1 — APPROVED** [review]

**Comment by aantn** [comment]
@NeonSludge thanks for the contribution! If you'd like some Robusta swag please reach out Maayan on our [Slack community](https://bit.ly/robusta-slack) and we'll send something your way.

**Pull request merged**